setup.cfg

```diff
@@ -22,7 +22,7 @@ packages = find:
 install_requires = 
 	napari
 	napari-plugin-engine>=0.1.4
-	cellpose>0.6.3
+	cellpose>=3.0.7
```

Comment:
Given that nobody is watching this repo, you might consider a stricter pin here to avoid similar incompatibilities in the future